operator-common/src/main/java/io/strimzi/operator/common/Annotations.java

```diff
@@ -18,6 +18,7 @@
 
     public static final String STRIMZI_DOMAIN = "strimzi.io/";
     public static final String STRIMZI_LOGGING_ANNOTATION = STRIMZI_DOMAIN + "logging";
+    public static final String STRIMZI_DYNAMIC_LOGGING_ANNOTATION = STRIMZI_DOMAIN + "dynamic-logging";
```

Comment:
just curious, why should we have a different annotation to highlight that this is updated dynamically? Just because we haven't moved yet the others components to use the dynamic logging and we want to distinguish it? Does it mean that when all will be moved to use dynamic logging, we can revert back to use just "logging" annotation? I don't see great value to have "dynamic-" prefix.

Reply:
Not sure what name this annotation should have. Let me describe why need this. Let's say user is running a strimzi without the dyn. logging and does *not* have `monitorInterval` entry in the log4j2.properties file. CO would add it but it does not have any effect until the pod is rolled and the log4j2 is started with log4j2.properties containing this entry. So it is necessary to roll the pod once the dynamic logging change is available.

Reply:
Tat does not sound to me like a valid reason. When upgrading the CO, it will need to roll all pods to the newest image version. And that should also automatically roll it to use the `monitorInterval`, or? So I would say it is not needed really.

Reply:
In that case, I can remove it. Thanks.